docs/swagger/content.yaml

```diff
@@ -203,6 +203,13 @@ definitions:
         type: object
         $ref: "#/definitions/Topics"
 
+  ContentForm:
```

Comment:
Why not integrate to `ContentMetadata` and introduce a new `Object`?

Reply:
answered here https://github.com/humhub/rest/pull/86#discussion_r851090851